src/logplex_api.erl

```diff
@@ -390,24 +390,18 @@ handlers() ->
                                             [What]),
                         json_error(422, Err);
                     {valid, _, URI} ->
-                        case logplex_channel:can_add_drain(ChannelId) of
```

Comment:
Hey - it looks like you removed the limit on number of drains per channel (5) via logplex_channel:can_add_drain. Was this intentional? Or is this called in some place I'm not seeing?